spec/unit/comments_spec.rb

```diff
@@ -210,4 +210,47 @@
       expect(resource.comments?).to eq false
     end
   end
+
+  describe "ActiveAdmin::Comment registration" do
+    let(:namespace) { application.namespace(namespace_name) }
+    let(:enable_resource_comment) { false }
+
+    before do
+      namespace.comments = enable_comments
+      resource = ActiveAdmin::Resource.new(namespace, Post)
+      resource.comments = enable_resource_comment
+      namespace.resources.add(resource)
+
+      # Manually trigger the before_load, after_load event
+      ActiveSupport::Notifications.instrument ActiveAdmin::Application::BeforeLoadEvent, { active_admin_application: application }
+      ActiveSupport::Notifications.instrument ActiveAdmin::Application::AfterLoadEvent, { active_admin_application: application }
+    end
+
+    context "when namespace.comments is true" do
+      let(:namespace_name) { :admin_with_comments }
+      let(:enable_comments) { true }
+
+      it "registers ActiveAdmin::Comment" do
+        expect(defined?(AdminWithComments::CommentsController)).to be_truthy
```

Comment:
is there a possibility that `defined?(AdminWithComments::CommentsController)` returns something different than true or false?

I would like to understand why this is checking for `be_truthy` instead of `be true`

Same for other similar specs

Reply:
`defined?` returns nil or string, like `"constants"`.
https://docs.ruby-lang.org/ja/latest/doc/spec=2fdef.html#defined

I should have use `be_nil` or something instead of ambiguous one. Let me fix it.

Reply:
Thanks for the clarification, I was wrongly assuming that all predicates return booleans, but I was probably misleaded by some recent cops enforcing this behavior.

Yes, `be_nil` maybe better, but in this case `be_truthy` is fine for me